src/aqa.ts

```diff
@@ -73,23 +74,46 @@ async function run(): Promise<void> {
     if (sdkdir === '') {
       sdkdir = process.cwd()
     }
-    await runaqa.runaqaTest(
-      version,
-      jdksource,
-      customizedSdkUrl,
-      sdkdir,
-      buildList,
-      target,
-      customTarget,
-      aqatestsRepo,
-      openj9Repo,
-      tkgRepo,
-      vendorTestParams,
-      aqasystemtestsRepo
-    )
+    if(runParallel === 'true'){
+
+      await runaqa.setupParallelEnv(
```

Comment:
Does setupParallelEnv eventually call runaqaTest?  If so, is there a reason we didn't add an extra runInParallel boolean argument to runaqaTest instead?

Reply:
It does not. The way we achieve parallelism in GitHub is to have one runner generate the `parallelList.mk` for the specific suite of tests cases. This file is uploaded to the workflow as an artifact. After which we spawn several other machines up which download the `parallelList.mk` with a specific list (e.g., testList_0) for which they are to run their tests. 

See here: https://github.com/j-braley/aqafer/actions/runs/2793593197 under "artifacts". 

The `aqafer` GH action contains the logic for downloading/running the test cases in parallel using a `strategy: matrix`